documentation/spring-boot-docs/src/docs/antora/modules/reference/pages/data/nosql.adoc

```diff
@@ -18,7 +18,12 @@ You can make use of the other projects, but you must configure them yourself.
 See the appropriate reference documentation at {url-spring-data-site}.
 
 Spring Boot also provides auto-configuration for the InfluxDB client but it is deprecated in favor of https://github.com/influxdata/influxdb-client-java[the new InfluxDB Java client] that provides its own Spring Boot integration.
+[WARNING]
```

Comment:
Please revert the change to the doc. We really only intend to document the property here.